src/app/pages/questions/components/toolbar/toolbar.component.html

```diff
@@ -1,7 +1,10 @@
-<ion-toolbar>
+<ion-toolbar mode="ios">
   <div class="toolbar-progress">
     <div class="toolbar-progress-bar" [style.width]="progress + '%'"></div>
```

Comment:
so we will show both a progress bar and progress counter?

Reply:
Yup it would show both if the progress counter is enabled.